tests/core/conftest.py

```diff
@@ -269,3 +347,55 @@ def protect_response(r):
     if r.status_code >= 300:
         message = 'Server responded with {r.status_code}\nbody:\n{r.text}'
         raise ValueError(message)
+
+
+def cleanup_dind(admin_mc, cluster, cluster_file):
+    admin_mc.client.delete(cluster)
+    remove_cluster(cluster_file)
+
+
+def create_kubeconfig(dind_cc, headers):
+    # wait for permissions to sync
+    time.sleep(1)
+
+    # request cluster scoped kubeconfig
+    KUBE_URL = BASE_URL + \
```

Comment:
This should use the client, you can get actions off `dind_cc.cluster.actions.<action_name>`